src/core/ext/filters/client_channel/lb_policy/pick_first/pick_first.cc

```diff
@@ -62,10 +62,9 @@ namespace grpc_core {
 
 TraceFlag grpc_lb_pick_first_trace(false, "pick_first");
 
-// TODO(eostroukhov): Remove once this feature is no longer experimental.
 bool ShufflePickFirstEnabled() {
```

Comment:
Let's just remove this completely.  If it's passing interop tests, we no longer need the knob.